### ISSUE TITLE
Question: how to receive a payload after connected to a websocket

### ISSUE THREAD

**Issue by adam-jr** [vic/apollo-phoenix-websocket#16]
I can see how to make a query or mutation from the client, but say my app's back end wants to send information in the same shape of a graphql response, but it has not been initiated by a query from the front end.

Is there any way to receive a payload like that through the channel and then insert it into Apollo's store?

**Comment by vic** [comment]
Hey, not sure I understand your use case. As far as I can tell, it's not a wise idea to try messing with the Apollo store, and if I'm understanding you well, you want to receive some data from the socket and update the store. 

Currently, you can give APOW (apollo-phoenix-websocket) an already initialized Phoenix socket, see the README, so it wont create another one. You can also as per Apollo documentation, give apollo the store you want it to use. 

I've already implemented some preliminar support for websocket subscriptions, that would be I guess, the best way to achieve what you want: fire an apollo subscription and let the server send data to apollo via the websocket. If you want to a look at this, see the #9 thread, there's an example chat application using it.

**Comment by adam-jr** [comment]
I think I see. I'm now doing something like this:

```
let socket;

const networkInterface = createNetworkInterface({
  uri: SOCKET_URL,
  channel: {
    topic: 'gql:query'
  },
  Socket: (url, options) => {
    socket = new PhoenixSocket(url, options);
    return socket;
  }
});
```

Advice taken on not modifying the Apollo store directly. Now I'm taking the socket and connecting to another channel that will be used to listen for events that will trigger a query... while I try to wrap my head around subscriptions.

**Comment by vic** [comment]
Hey @atom-r I've just released version 0.6.0 which subscriptions support (as in *beta*, just like in Absinthe). If you want to use subscriptions now is the time, checkout this example app https://github.com/vic/verily 

I'm closing this issue now, as the long awaited subscription branch has been merged.
